src/core/hle/service/sockets/nsd.cpp

```diff
@@ -40,6 +45,55 @@ NSD::NSD(Core::System& system_, const char* name) : ServiceFramework{system_, na
     RegisterHandlers(functions);
 }
 
+static ResultVal<std::string> ResolveImpl(const std::string& fqdn_in) {
+    // The real implementation makes various substitutions.
+    // For now we just return the string as-is, which is good enough when not
+    // connecting to real Nintendo servers.
+    LOG_WARNING(Service, "(STUBBED) called({})", fqdn_in);
```

Comment:
```suggestion
    LOG_WARNING(Service, "(STUBBED) called, fqdn_in={}", fqdn_in);
```
to be consistent with our logging style